README.md

```diff
@@ -41,9 +41,9 @@ We are willing to answer questions.
 ## Getting Started
 
 ```javascript
-import vjsn from "zome-vjsn";
+import vueSpatialNavigation from "zome-vjsn";
```

Comment:
Why the change?

Reply:
I replaced `import vjsn` with `import vueSpatialNavigation` because its the currently exported name ([see here](https://github.com/Syncronet-APS/zome-vjsn/blob/80ff7c71dcd37e07d94ef7788006ceb1661dd8a1/lib/index.ts#L153))

Reply:
Oh alright I understood, my IDE was tricking me into thinking there was an error with just `vjsn` because this name wasn't defined anywhere. But since it is the default export we can name it whatever we want. I will revert the changes, sorry!